pip-requirements-doc

```diff
@@ -16,5 +16,3 @@ jplephem
 sphinx<1.7
```

Comment:
sphinx should be installed by sphinx-astropy as well, but probably there is no harm leaving it here, too (it would at least let us pin the version if needed)